pyopendoc/writer.py

```diff
@@ -75,6 +75,27 @@ def write_to_table(self, table_name, column, row, value, refresh=False, target=N
 
         span.text = value
 
+    def write_to_frame(self, frame_name, x_value, y_value, value=None, target=None, unit="in"):
+        """Writes to x, y coordinate of a specified frame.
+        Defaults unit to inches.
+
+        """
+        target_file = target if target else self.CONTENT_FILE
+        xml_file = self.get_file(target_file)
+        if xml_file.filetype != "XML":
+            raise TypeError("Not an XML file!")
+
+        for df in xml_file.root.findall(".//draw:frame", self.NAMESPACES):
+            frame = df.get("{%s}name" % self.NAMESPACES["draw"])
+            if frame == frame_name:
```

Comment:
When using a combination of for-loop and if-statement to find and do something to a single item within a collection, you'll want to use either a break statement (to exit the loop) or a return (to exit the function) to terminate immediately after you have found what you are looking for. It may or may not be particularly important in this case, but it is a good habit to get into.

(I'm assuming that you want just the one frame.)